lib/hanami/routing/http_router.rb

```diff
@@ -67,6 +67,7 @@ class HttpRouter < ::HttpRouter
       # @since 0.1.0
       # @api private
       def initialize(options = {}, &blk)
+        Hanami::Utils::Deprecation.new('options[:parsers] is deprecated and it will be removed in future versions')
```

Comment:
I think this message should be displayed only if `:parsers` is used. In addition, we have to say where is the reference of this option, something like this? 

`Hanami::Router options[:parsers]`

Reply:
Thank you @AlfonsoUceda I will change it